Raven.Database/Config/StronglyTypedRavenSettings.cs

```diff
@@ -244,6 +244,8 @@ public void Setup(int defaultMaxNumberOfItemsToIndexInSingleBatch, int defaultIn
             MaxStepsForScript = new IntegerSetting(settings["Raven/MaxStepsForScript"], 10 * 1000);
             AdditionalStepsForScriptBasedOnDocumentSize = new IntegerSetting(settings["Raven/AdditionalStepsForScriptBasedOnDocumentSize"], 5);
 
+            SkipConsistencyCheck = new BooleanSetting("Raven/Storage/SkipConsistencyCheck", false);
```

Comment:
this should be `new BooleanSetting(settings["Raven/Storage/SkipConsistencyCheck"], false);`